server/src/main/java/org/apache/iotdb/db/metadata/idtable/entry/DeviceEntry.java

```diff
@@ -32,9 +32,18 @@
 
   boolean isAligned;
 
+  // for manages flush time
```

Comment:
```suggestion
  // for managing flush time
```

Add javadoc for key and value, time partition -> flushTime

Reply:
fixed~